### DISCUSSION TITLE
how to check the log4j version running in the java runtime

### DISCUSSION THREAD

**Discussion by pantdeepak** [apache/logging-log4j2#1924] [General]
Up until version 2.20.0, I was able to use following method to check the version #.
org.apache.logging.log4j.LogManager.class.getPackage().getImplementationVersion()

Starting version 2.21.1 the above method returns null as I could not find "Implementation-Version" entry in the MANIFEST.MF file.

**Comment by rgoers** [comment]
@vy Can you please modify the CI build to fix this? Note that this should be converted to a bug since it breaks currently supported functionality.

**Comment by vy** [comment]
The discussion has been moved to #1923.
